build.bash

```diff
@@ -264,15 +270,14 @@ if [ "$hw_platform" == "pi-raspios32" ] || [ "$hw_platform" == "pi-raspios64beta
   cp $sourcefolder/openhabian-installer.service $buildfolder/root/etc/systemd/system/
   ln -s $buildfolder/root/etc/systemd/system/openhabian-installer.service $buildfolder/root/etc/systemd/system/multi-user.target.wants/openhabian-installer.service
 
-  echo_process "Setting default runlevel multiuser.target... "
-  ( # open a subshell, returns to current dir automatically
-    cd $buildfolder/root/etc/systemd/system/ || exit 1
-    rm -rf default.target
-    ln -s ../../../lib/systemd/system/multi-user.target default.target
-    # disable autologin
-    rm -f $buildfolder/root/etc/systemd/system/getty@tty1.service.d/autologin.conf
-  )
+  echo_process "Setting default runlevel multiuser.target and disabling autologin... "
+  rm -rf $buildfolder/root/etc/systemd/system/default.target
+  ln -s $buildfolder/lib/systemd/system/multi-user.target default.target
+  rm -f $buildfolder/root/etc/systemd/system/getty@tty1.service.d/autologin.conf
 
+  echo_process "Copying openHABian files to installation... "
+  # '/opt/openhabian/' must be in place for offline install to work.
+  if ! [[ -d $buildfolder/root/opt/openhabian ]]; then git clone "${repositoryurl:-https://github.com/openhab/openhabian.git}" $buildfolder/root/opt/openhabian &> /dev/null; fi
```

Comment:
this actually breaks dev-git and dev-url option, right?
If you want to implement this here pls look at `clone_string`...

Reply:
Nope actually, it still works we change the variables before so as far as I can tell looking at the logic of the script it still functions properly.